app/controllers/episodes_controller.rb

```diff
@@ -8,4 +8,34 @@ def index
   def show
     @episode = Episode.find(params[:id])
   end
+
+  def update_youtube_info
+    @episode = Episode.find(params[:id])
+    YoutubeService.new(@episode).save_all
+    redirect_to episode_path
+  end
+
+  def online
+    @episode = Episode.find(params[:id])
+    @timecodes = @episode.timecodes
```

Comment:
can you get `@episode.timecodes` in online view?

Reply:
No, this command is still a request to database :) You can check it by seeing server logs when the page is loading. All the requests to database should be in controller as far as I know.